From c04aab9b1ba551b65c9e51de094ee3864a45f193 Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Sat, 16 Mar 2024 13:07:15 +0200
Subject: [PATCH] [SW-156330] gaudi3: increase ppboot size

As preparation for addition of TPM feature, gaudi3
ppboot size is increased from 16kB to 17Kb. TPM
feature addition increases the size of ppboot by
around 5.2kb. Current ppboot size is found to be
around 11.5kB.

Given that secondary image support is disabled,
PCIe link up time with this increase of ppboot
image size is calculated to be 77ms and well
within the 100ms limit.

Change-Id: Iad96792a00a46d86f4644aabeacf617a27809cde
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/arch/gaudi3/arc/Makefile b/arch/gaudi3/arc/Makefile
index fb66863..9b5ee54 100644
--- a/arch/gaudi3/arc/Makefile
+++ b/arch/gaudi3/arc/Makefile
@@ -1,11 +1,7 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 include $(HL_DEV_ARCH_COMMON)/pcie_phy_fw.mk
 
-ifeq ($(HL_TARGET),pldm)
-	PPBOOT_MAX_SIZE := 17408
-else
-	PPBOOT_MAX_SIZE := 16384
-endif
+PPBOOT_MAX_SIZE := 17408
 
 ifeq ($(HL_TARGET),onix)
 	LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports_onix.elf
diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index cb80322..f53777e 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -8,11 +8,7 @@
 
 #include <sizes.h>
 
-#ifdef TARGET_IS_PLDM
 #define PPBOOT_SIZE			0x4400		/* 17KB */
-#else
-#define PPBOOT_SIZE			0x4000		/* 16KB */
-#endif
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 
 #define HL_FW_BOOT_PTRS_BASE		0x60000
